examples/cursor_ray.py

```diff
@@ -11,18 +11,21 @@
 # data to depict an empty volume, its bounding box and points along a ray
 # through the volume
 volume = np.zeros(shape=(sidelength_data, sidelength_data, sidelength_data))
-bounding_box = np.array(
-    [
-        [0, 0, 0],
-        [1, 0, 0],
-        [0, 1, 0],
-        [1, 1, 0],
-        [0, 0, 1],
-        [1, 0, 1],
-        [0, 1, 1],
-        [1, 1, 1],
-    ]
-) * sidelength_data
+bounding_box = (
```

Comment:
my black also corrects these... but this bothers my sense of PR granularity :joy:  (i.e. when we look back to see what PR modified a file, if we're going to do a pure formatting change on an unrelated file, I'd like the PR to be something like "black reformat", not "add hook spec")

can you revert this file, and other un-related file formats?

do you have a `--all-files` setting in your `pre-commit` or black settings somehow?  I'm curious why this would have been modified at all?

Reply:
I am using the pre-commit config without any modifications. I will check which files were just formatted and undo them.